enhancing/enhancer.js

```diff
@@ -2,21 +2,44 @@ module.exports = {
   succeed,
   fail,
   repair,
-  get,
+  get
 };
 
 function succeed(item) {
-  return { ...item };
+  let newItem = {
+    name: item.name,
```

Comment:
Could have just spread in the item and then just changed the single property you wanted to change.